src/clj/rems/workflow/dynamic.clj

```diff
@@ -772,11 +871,15 @@
      (catch RuntimeException e#
        (ex-data e#))))
 
+;; TODO rename to be test-specific
```

Comment:
Or move the high-level tests (i.e. tests for commands and calculate-permissions) to another file.

Reply:
yeah I guess the time is ripe for that